wodles/aws/tests/test_cloudtrail.py

```diff
@@ -7,11 +7,12 @@
 import sys
 from unittest.mock import patch
 
+import wodles.aws.constants
```

Comment:
Is there any special reason why this import is different than the other tests?

Reply:
Fixed in commit 51691977a6ebae956149241cd801cb68d2a8cc0e.